source/boundary_velocity/gplates.cc

```diff
@@ -727,12 +727,12 @@ namespace aspect
 
       if ((this->get_time() - first_data_file_model_time >= 0.0) && (this->get_geometry_model().depth(position) <= lithosphere_thickness + magic_number))
         {
-          const Tensor<1,dim> data = lookup->surface_velocity(position);
+          const Tensor<1,dim> data = scale_factor * lookup->surface_velocity(position);
```

Comment:
Could we rename this `velocity_scaling_factor`? I think that would make it clearer what this factor should be applied to.

Reply:
I changed it but unfortunately it's still called "Scale factor" in the input file. Given that the parameter did not do anything in the past, do you want me to change that too?